src/transformers/models/ctrl/modeling_ctrl.py

```diff
@@ -456,7 +456,9 @@ def forward(
 
         inputs_embeds *= np.sqrt(self.d_model_size)
 
-        pos_embeds = self.pos_encoding[position_ids, :].to(device)
+        # `self.pos_encoding` won't be sent to the correct device along the model, so we do it manually.
+        self.pos_encoding = self.pos_encoding.to(device)
+        pos_embeds = self.pos_encoding.to(device)[position_ids, :].to(device)
```

Comment:
```suggestion
        pos_embeds = self.pos_encoding[position_ids, :]
```
A bit of an overkill here :sweat_smile: